ui/packages/catalog-ui-search/src/main/webapp/component/query-title/query-title.view.js

```diff
@@ -84,6 +84,7 @@ module.exports = Marionette.LayoutView.extend({
   },
   handleTitleUpdate() {
     this.$el.find('input').val(this.model.get('title'))
+    this.$el.find('.button-title').html(this.getSearchTitle() + zeroWidthSpace)
```

Comment:
❓ Would you be willing to pull up this title change/update code into a separate function and just call it from the (at least two that I see) places in the code? That way we encapsulate the change.